supertokens_python/supertokens.py

```diff
@@ -314,7 +314,7 @@ async def get_user_count(  # pylint: disable=no-self-use
     async def delete_user(  # pylint: disable=no-self-use
         self,
         user_id: str,
-        user_context: Optional[Dict[str, Any]],
+        user_context: Optional[Dict[str, Any]] = None,
```

Comment:
once again, internal functions should not default to setting user_context to None. All functions in this file are internal